BrainPortal/app/models/data_provider.rb

```diff
@@ -1360,6 +1360,22 @@ def self.rsync_ignore_patterns #:nodoc:
     @ig_patterns ||= RemoteResource.current_resource.dp_ignore_patterns || []
   end
 
+  # Some Bourreaux systems are configured with disk allocations where files older than N days are erased automatically.
+  # To prevent such system from deleting the top-level directories for the DP_Cache, and some cbrain-specific files,
+  # the system can reset their timestamps using the touch command.
+  #
+  # - the +DataProvider+ cache dir
+  # - the +DP_Cache_Key.md5+ and
+  # - +DP_Cache_Rev.id+ located in that cache dir
+  def self.system_touch
```

Comment:
The method name `system_touch` is way too general. The method touches two special config files related to the cache subsystem. In general, in the DataProvider class, all methods that deal with the cache subsystem have "cache_" in their names. So please rename this

```ruby
def self.touch_cache_md5
```

and place it after `create_cache_md5` where it belongs.